apps/nextra/pages/en/build/smart-contracts/book/enums.mdx

```diff
@@ -0,0 +1,214 @@
+# Enums
```

Comment:
In the linked nextra preview, I would have expected "Enums" to appear in the left bar under "Basic Concepts", perhaps right under "Structs and Resources".

I only see it under "Reference", just below "Coding Conventions".